swagger.yaml

```diff
@@ -138,6 +189,16 @@ definitions:
       href:
         type: string
         description: "The URL to the resource"
+  TopicLink:
+    type: object
+    description: "A link to the topic for this content"
+    properties:
+      href:
+        type: string
+        description: "The URL to the topic"
+      id:
+        type: string
+        description: "The id for the topic that this content is associated with"
   Count:
     type: string
```

Comment:
I noticed that the limit, offset and count are defined as strings (`type : string`), I know it was already like this before this PR, but could you replace them for `type: integer` please?

Reply:
Yes, now fixed